src/kixi/heimdall/handler.clj

```diff
@@ -5,24 +5,21 @@
             [ring.middleware.keyword-params :refer [wrap-keyword-params]]
             [kixi.heimdall.application :as app]
             [kixi.heimdall.service :as service]
+            [kixi.heimdall.util :as util]
             [taoensso.timbre :as log]
             [clojure.java.io :as io]
             [clojure.edn :as edn]
-            [environ.core :refer [env]]
-            ))
+            [environ.core :refer [env]]))
 
 (defn get-config
   [f]
   (edn/read-string (slurp f)))
 
 (def auth-config
-  (let [f (io/file (System/getProperty "user.home")
-                   ".heimdall.auth-conf.edn")]
-    (try (get-config f)
-         (catch java.io.FileNotFoundException e
-           (if-let [env-file (:auth-conf env)]
-             (get-config (io/resource env-file))
-             (throw e))))))
+  (-> (if (util/file-exists? (System/getProperty "user.home") ".heimdall.auth-conf.edn")
+        (io/file (System/getProperty "user.home" ".heimdall.auth-conf.edn"))
```

Comment:
this will just return the home directory, not the ~/.heimdall.auth-conf.edn

```user> (System/getProperty "user.home" ".bashrc")
"/home/bld"

user> (clojure.java.io/file (System/getProperty "user.home") ".bashrc")
#object[java.io.File 0x457f0351 "/home/bld/.bashrc"]
```